### ISSUE TITLE
[auth] add email/password prompt for `visibility: "protected"` in `netzoAuth` plugin

### ISSUE THREAD

**Issue by miguelrk** [netzo/netzo#43]
**TLDR**: Use `window.prompt` to prompt for `email` and `password` when `visibility: "protected"` in `netzoAuth` plugin like below:

**IMPORTANT**: the deno_kv_oauth fresh plugin was added to fresh in 1.5 and removed from deno_kv_oauth, but then removed from fresh in 1.5.1. Here is the [old plugin implementation](https://github.com/denoland/deno_kv_oauth/commit/a475ea35c0c2311995a7822b28fd594c8aa62e72) from deno_kv_oauth for reference and [the one ported to fresh 1.5](https://github.com/denoland/fresh/commit/73aaf84fad1310b4c9577b49b441923e69beb170) for reference.

![image](https://github.com/netzo/netzo/assets/18708856/54136ba7-d3d5-4138-980e-82967c4a7756)

**Comment by miguelrk** [comment]
Closing as no longer relevant
